src/main/java/org/folio/rest/impl/PutOrdersByIdHelper.java

```diff
@@ -109,7 +109,13 @@ public void updateOrder(String orderId, CompositePurchaseOrder compPO) {
   }
 
   private boolean isTransitionToOpen(CompositePurchaseOrder compPO, JsonObject poFromStorage) {
-    WorkflowStatus currentStatus = WorkflowStatus.fromValue(poFromStorage.getString("workflow_status"));
+    // MODORDERS-150 Workflow status might be null. Use default as 'Pending' in such case
+    WorkflowStatus currentStatus;
+    if (StringUtils.isNotEmpty(poFromStorage.getString("workflow_status"))) {
```

Comment:
Wouldn't it be better to set the default "Pending" for "workflow_status" in the "purchase_order" scheme instead of this check?

Reply:
Thanks! The https://github.com/folio-org/acq-models/pull/85 created.